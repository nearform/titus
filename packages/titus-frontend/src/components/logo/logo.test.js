import React from 'react'
import { render, cleanup } from '@testing-library/react'

import Logo from '.'

describe('<Logo />', () => {
  afterEach(cleanup)

  it('should render correctly', () => {
    const { asFragment } = render(<Logo />)
    expect(asFragment()).toMatchSnapshot()
  })
})
