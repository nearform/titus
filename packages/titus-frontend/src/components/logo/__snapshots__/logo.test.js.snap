// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render correctly 1`] = `
<DocumentFragment>
  <img
    alt="Titus logo"
    class="login-logo"
    src="img/Accel_Logo_Titus.svg"
  />
</DocumentFragment>
`;
